src/fides/core/config/redis_settings.py

```diff
@@ -46,8 +84,8 @@ def assemble_connection_url(
             connection_protocol = "rediss"
             cert_reqs = values.get("ssl_cert_reqs", "none")
             params = f"?ssl_cert_reqs={quote_plus(cert_reqs)}"
-
-        return f"{connection_protocol}://{quote_plus(values.get('user', ''))}:{quote_plus(values.get('password', ''))}@{values.get('host', '')}:{values.get('port', '')}/{db_index}{params}"
+        connection_url = f"{connection_protocol}://{quote_plus(values.get('user', ''))}:{quote_plus(values.get('password', ''))}@{values.get('host', '')}:{values.get('port', '')}/{db_index}{params}"
+        return connection_url
```

Comment:
I flip back and forth on this it seems, is there any consensus (or strong opinion) on setting the value as a param and then returning vs inline?

Reply:
I _personally_ feel it is cleaner to set a variable and then return it, for a few reasons:

1. easier to further manipulate that value, print it, etc.
2. Each line has a single purpose. This line creates a var, this one returns a var, etc.

I'm not the gatekeeper of all python style though, that's just what works best for me

Reply:
Sounds good, thanks for heads up! I do that similarly except in very straight forward cases. I've also noticed I almost need to set it prior to returning with certain things like SqlAlchemy